test/integration/content-explorer/ContentExplorer.e2e.test.js

```diff
@@ -0,0 +1,207 @@
+// <reference types="Cypress" />
+const helpers = {
+    load(additionalProps = { canUpload: true, permissions: { canUpload: true } }) {
+        cy.visit('/Elements/ContentExplorer', {
+            onBeforeLoad: contentWindow => {
+                contentWindow.PROPS = additionalProps;
+            },
+        });
+    },
+    getRow(rowNum) {
+        return cy.getByDataPreview('ContentExplorer').find(`.bce-item-row-${rowNum}`);
+    },
+    checkRows(rowNum) {
+        cy.getByDataPreview('ContentExplorer')
+            .find('.ReactVirtualized__Table')
+            .then(table => {
+                const totalRows = table[0].getAttribute('aria-rowcount');
+
+                for (let i = 0; i < totalRows; i += 1) {
+                    if (i !== rowNum) {
+                        this.getRow(i).should('not.have.class', 'bce-item-row-selected');
+                    }
+                }
+                if (rowNum >= 0) {
+                    this.getRow(rowNum).should('have.class', 'bce-item-row-selected');
+                }
+            });
+    },
+    selectRow(rowNum) {
+        this.getRow(rowNum)
+            .as('row')
+            .click()
+            .should('have.class', 'bce-item-row-selected');
```

Comment:
This class name appears more than twice, so should probably be a constant or helper method.